.github/workflows/codecov.yaml

```diff
@@ -13,22 +13,16 @@ concurrency:
 
 jobs:
   coverage:
-    name: Code Coverage
```

Comment:
Keep the name?

Reply:
it already shows the name of the workflow from the top of the file